site/pages/api/admin/add-user.api.ts

```diff
@@ -35,11 +56,24 @@ const addUser = async (req: NextApiRequest, res: NextApiResponse) => {
             return;
         }
 
-        await createUser(validatedBody.data);
+        if (validatedBody.data.group === UserGroups.operators) {
+            const nocCodes = validatedBody.data.operatorNocCodes?.map((operator) => operator.nocCode) ?? [];
+            const operatorAttribute: AttributeType[] = [
+                {
+                    Name: "custom:nocCodes",
+                    Value: nocCodes.toString(),
```

Comment:
`nocCodes.join(",")` is probably the more idiomatic way of concatenating the strings here

Reply:
done